backend/climateconnect_api/management/commands/create_test_data.py

```diff
@@ -135,21 +136,30 @@ def create_project_test_data(number_of_rows: int):
         url_slug = name.replace(" ", "")
 
         if not Project.objects.filter(name=name).exists():
+            # Buffer some start date to create more
+            # sense of older projects. This originated out of
+            # fixing a bug around the "timeago" utility, where
+            # timeago would round up to 2 years, when a project
+            # is only 1 year and 1 day old:
+            over_a_year_ago = timezone.now() - timezone.timedelta(days = 366)
```

Comment:
@ddhanesha I added this as the default to create more variance in our test data. WDYT?

Reply:
Sounds good! The more different we make our test data, the easier it will be to find missing edge cases while developing locally